bpf/lbmap/ipv4.go

```diff
@@ -38,6 +38,11 @@ var (
 		int(unsafe.Sizeof(RevNat4Key{})),
 		int(unsafe.Sizeof(RevNat4Value{})),
 		maxEntries)
+	RRSeq4Map = bpf.NewMap("cilium_lb4_rr_seq",
+		bpf.MapTypeHash,
+		int(unsafe.Sizeof(Service4Key{})),
+		int(unsafe.Sizeof(ServiceRRSeq{})),
```

Comment:
I do see a point in changing the names in the sense that a grep would tell me which of the Service4Keys are used for RRSeq. I will rename it.